docs/governance/decred-constitution.md

```diff
@@ -1,48 +1,41 @@
+- For a rule change proposal to be approved, at least 75% of the tickets not set to Abstain must vote Yes. If this requirement is met, and a quorum of 10% tickets voting Yes or No is achieved, then the rule change will be activated 8,064 blocks (~4 weeks) later.
 
 ------
 
 ## Project Governance and Funding
 
 * Sustainability and longevity require that 10% of all block rewards be given to a project Treasury fund, administered by a development organization.
-* The initial development organization is *Decred Holdings Group* LLC ("DHG"), a Nevis LLC that is responsible for funding work related to the development of the project, such as software development, infrastructure, and awareness.
-* DHG shall only fund work that adheres to the guiding principles.
+* The initial development organization is *Decred Holdings Group* LLC ("DHG"), a Nevis LLC that is responsible for funding work related to the development of the project, such as software development, infrastructure, and awareness. DHG shall comply with all relevant bodies of law in the jurisdictions where applicable, such as embargoes and other trade sanctions.
 * The project aims to transition to a Decentralized Autonomous Entity which controls the disbursement of Treasury funds.
-* Politeia proposals are the mechanism through which stakeholders make policy decisions and approve programs of work. 
-* Politeia proposals must be approved by at least 60% of the tickets that vote, and at least 20% of the eligible tickets must participate in the vote. The default voting period is 2016 blocks (~1 week).
-* Decred contractors are individuals and corporations who work on the project, under an agreement that they are compensated for their work. 
-* Decred contractors are autonomous, they cannot be managed directly by stakeholders. Through Politeia, stakeholders can approve or reject programs of work as performed by specific contractors, but they cannot mandate that workers take certain actions. 
-* The Decred contractor collective is a self-regulating entity, wherein cells working on specific aspects resolve disputes and add new members independently.
-* As described in the approved [Decred Contractor Clearance proposal](https://proposals.decred.org/proposals/fa38a3593d9a3f6cb2478a24c25114f5097c572f6dadf24c78bb521ed10992a4), disputes that cannot be satisfactorily resolved within a team can be escalated to an all contractor vote - once the contractor management system which will facilitate this is operational. If necessary, unresolved matters could be further escalated to a stakeholder vote as the ultimate decision-making authority. 
+* For as long as DHG has responsibility for managing the Treasury (i.e. until control is fully decentralized), it holds a veto that can be exercised if it is felt that following through with a decision would put the project, the DHG corporate entity or DHG's managers in jeopardy. 
```

Comment:
How would this veto apply? When can the veto be overruled? What does it mean to be "in jeopardy?"

May want some exemplary situations of "in jeopardy" to clarify the phrase.

Reply:
While the treasury is controlled manually by DHG, the people involved can decide not to make transactions if they feel that veto is justified. This will likely change in the not too distant future, so it makes sense to amend the constitution with details of whatever new approach stakeholders approve to transition to.

Reply:
I see. Makes more sense.

Reply:
Also, how often are you expecting the Constitution to be amended?